mmv1/templates/terraform/constants/resource_dns_resource_record_set.go.erb

```diff
@@ -11,9 +18,29 @@ func rrdatasDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return false
 }
 
-func ipv6AddressforDnsDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
-	oldIp := net.ParseIP(old)
-	newIp := net.ParseIP(new)
+func ipv6AddressforDnsDiffSuppress(x, y []string, _ *schema.ResourceData) bool {
+	if len(x) != len(y) {
```

Comment:
different length rrdatas could technically result in the same value, if there's duplicates. But this also doesn't have to handled on the TF layer either (the API should dedupe).

Reply:
This intends to handle the case when the record count is increased or decreased. Shouldn't users be able to add or remove the IPs?

Reply:
yes, but my point was more than a modified length doesn't imply a diff. I could add a redundant IP to y, for example, which would mean that the end result is the same (same set of IPs), but the count would be different.

Reply:
Got it